features/upgradables.feature

```diff
@@ -203,6 +203,6 @@ Feature: Manage WordPress themes and plugins
     And the return code should be 1
 
     Examples:
-      | type   | type_name | item                    | item_title              | version | zip_file                                                              | file_to_check                                                    |
+      | type   | type_name | item                    | item_title              | version | zip_file                                                               | file_to_check                                                     |
```

Comment:
What's the deal with this very special space? :)

Reply:
Note: I think this can all be compressed anyway, the free space in-between should not matter.

Reply:
Huh, I thought the space was off so I added it for better visual alignment locally 🤔

Reply:
Makes it a bit nicer to look at during local development when everything is nicely aligned:

<img width="1066" alt="Screenshot 2023-11-10 at 12 06 16" src="https://github.com/wp-cli/extension-command/assets/841956/b4b42257-455d-4de6-8c87-9d711d301fd6">

Reply:
I think that depends on the actual version number, though. But I'm fine with merging it like this.